runtime/doc/options.txt

```diff
@@ -4333,6 +4333,20 @@ A jump table for the options with a short description can be found at |Q_op|.
 	In its simplest form the value is just one font name.
 	See |gui-font| for the details.
 
+					*'guidarkmode'*
+'guidarkmode'		string	(default "automatic")
+			global
+			{only available when compiled with GUI enabled}
+	This option specifies whether Vim GUI elements should be drawn using a
+	dark color scheme if available.
```

Comment:
color scheme has a very distinct meaning in Vim. Is this really a Vim color scheme or something else?